tools/proto-js-plugin/src/main/java/io/spine/js/generate/parse/ExportStandardParsersMap.java

```diff
@@ -58,7 +62,7 @@
  * <p>The parsers may be used to parse JSON via their {@code parse(value)} method.
  */
 @SuppressWarnings("OverlyCoupledClass") // Dependencies for the listed Protobuf types.
-public final class ProtoParsersGenerator extends JsCodeGenerator {
+public final class ExportStandardParsersMap implements Snippet {
```

Comment:
I don't understand calling this snipped ending with `Map`. Will we need to rename the class if it stops generating a `Map` but generates a JS class instead?